pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -80,7 +80,7 @@ func checkIfUserPartOfHyperVAdmins() (bool, error) {
 	@([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole($sid)`
 	stdOut, _, _ := powershell.Execute(checkIfMemberOfHyperVAdmins)
 	if !strings.Contains(stdOut, "True") {
-		return false, errors.New("User is not a member of the Hyper-V administrators group")
+		return false, errors.New("User is not a member of the Hyper-V administrators group.")
```

Comment:
Not sure we want '.' at the end of messages (this was pointed out to me as part of another review).

Reply:
Fixed. It is a Go convention